### PR TITLE
Do not require an account id for importing addresses

### PR DESCRIPTION
**Pull request by KtorZ**
## Description

Backport of https://github.com/input-output-hk/cardano-wallet/pull/320

Actually, we were sort-of ignoring it already and the endpoint was
misleading.  Cardano addresses embed their corresponding account index
inside them, therefore, when importing addresses, we only care about the
root master key (and therefore, the root id). The account ix comes from
the address anyway

## Type of change

- [ ] 🐞 Bug fix (non-breaking change which fixes an issue)
- [ ] 🛠 New feature (non-breaking change which adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🏭 Refactoring that does not change existing functionality but does improve things like code readability, structure etc
- [ ] 🔨 New or improved tests for existing code
- [ ] ⛑ git-flow chore (backport, hotfix, etc)

## Developer checklist

- [ ] I have read the [style guide](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/style-guide.md) document, and my code follows the code style of this project.
- [ ] If my code deals with exceptions, it follows the [guidelines](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/exceptions.md).
- [ ] I have updated any documentation accordingly, if needed. Documentation changes can be reflected in opening a PR on [cardanodocs.com](https://github.com/input-output-hk/cardanodocs.com), amending the inline [Haddock](https://www.haskell.org/haddock/) comments, any relevant README file or one of the document listed in the [docs](https://github.com/input-output-hk/cardano-sl/tree/develop/docs) directory.
- [ ] CHANGELOG entry has been added and is linked to the correct PR on GitHub.

## Testing checklist

- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## How to merge

Send the message `bors r+` to merge this PR. For more information, see
[`docs/how-to/bors.md`](https://github.com/input-output-hk/cardano-sl/blob/develop/docs/how-to/bors.md).

### CODE REVIEW

File: wallet/src/Cardano/Wallet/API/V1/Swagger.hs [hunk 1]
```diff
@@ -990,7 +990,7 @@ derivation, it is unlikely that the same addresses will be generated on two
 different node instances. However, some API users may wish to preserve unused
 addresses between different instances of the wallet backend.
 
-To enable this, the wallet backend provides an endpoint ([`POST /api/v1/wallets/{{walletId}}/accounts/{{accountId}/addresses`](#tag/Addresses%2Fpaths%2F~1api~1v1~1wallets~1{walletId}~1accounts~1{accountId}~1addresses%2Fpost))
+To enable this, the wallet backend provides an endpoint ([`POST /api/v1/wallets/{{walletId}}/accounts/{{accountId}/addresses`](#tag/Addresses%2Fpaths%2F~1api~1v1~1wallets~1{walletId}~1addresses%2Fpost))
```
Comment (piotr-iohk) [inline wallet/src/Cardano/Wallet/API/V1/Swagger.hs]: ups, just spotted. `/accounts/{{accountId}` needs to be removed.

Comment (KtorZ) [reply wallet/src/Cardano/Wallet/API/V1/Swagger.hs]: Erf. Good catch.

**Comment by KtorZ** [comment]
I don't like you either bors.

**Pull request merged**